lib/ansiblelint/__main__.py

```diff
@@ -56,6 +57,20 @@ def initialize_logger(level: int = 0) -> None:
     _logger.debug("Logging initialized to level %s", logging_level)
 
 
+def initalize_formatter_factory(options_list: Namespace):
+    """Apply options from command line arguments to the formatter_factory."""
+    formatter_factory: Any = formatters.Formatter
```

Comment:
This time declaration is probably unnecessary
```suggestion
    formatter_factory = formatters.Formatter
```

Reply:
But if it complains, use something like
```suggestion
    formatter_factory: Type[formatters.BaseFormatter] = formatters.Formatter
```

Reply:
(unless `TypeVar` is necessary, I haven't checked it)